index.html

```diff
@@ -420,6 +398,243 @@ <h2>
           lock that applies on its <a>top-level browsing context</a>. An
           orientation lock is an unordered set of <a>OrientationType</a>.
         </p>
+      </section>
+      <section>
+        <h2>
+          Default orientation
+        </h2>
+        <p>
+          A <a>document</a>'s <dfn>default orientation</dfn> is the set of
+          orientations to which the screen orientation is locked when it is not
+          explicitly locked by this API or any other means.
+        </p>
+        <p>
+          For the perspective of a <a>document</a>, locking to the <a>default
+          orientation</a> is equivalent to unlocking because it means that it
+          no longer has a lock applied. However, it does not mean that the
+          <a>default orientation</a> has to be <code>any</code>.
+        </p>
+      </section>
+    </section>
+    <section>
+      <h2>
+        Interactions with other specifications
+      </h2>
+      <p>
+        This section explains how this specification interacts with other
+        related specifications of the platform.
+      </p>
+      <section>
+        <h2>
+          Interaction with Fullscreen API
+        </h2>
+        <p>
+          As a <a data-lt="security conditions">security condition</a>, a user
```

Comment:
Ah, I'd forgotten that we now support automatic pluralization detection support in ReSpec. I just sent at pull request to enable it by default: https://github.com/w3c/respec/pull/2020

```suggestion
          As a <a>security condition</a>, a user
```